### PR TITLE
ActivityIndicator: add snackplayer/functional component with hooks

### PR DESCRIPTION
**Pull request by ronnyiscoding**
Open Issue: #1579

- Added snackplayer functional example for ActivityIndicator component

### CODE REVIEW

File: website/i18n/en.json [hunk 1]
```diff
@@ -341,9 +341,6 @@
       "toastandroid": {
         "title": "ToastAndroid"
       },
-      "toolbarandroid": {
```
Comment (Simek) [inline website/i18n/en.json]: @ronnyiscoding Same problem here.

Comment (ronnyiscoding) [reply website/i18n/en.json]: PR updated here https://github.com/facebook/react-native-website/pull/1590/commits/12166a4f8a16a7bd82ec553fb7675fd0886fe5d2 thanks!

Comment (Simek) [reply website/i18n/en.json]: Thank you for the changes. 

As @rachelnabors said (which apply for the both PRs) you should be updating:
* `master` docs (`docs/activityindicator.md`) 

not:

* `versioned` docs (`website/versioned_docs/version-0.61/activityindicator.md`)

Comment (ronnyiscoding) [reply website/i18n/en.json]: Ok, Thank you, PR updated at https://github.com/facebook/react-native-website/pull/1590/commits/51913e47619c9295d44ffe4543601b6335e2d541

**Comment by ronnyiscoding** [comment]
> Ach, you've don't this in the versioned docs! We need the file to be edited here:
> https://github.com/facebook/react-native-website/blob/master/docs/activityindicator.md

Thanks for the review, PR is updated please let me know if this is the right way to contribute 👍

**Pull request merged**